pkg/blockstorage/azure/client.go

```diff
@@ -20,27 +20,33 @@ package azure
 
 import (
 	"context"
-
-	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-03-01/compute"
-	"github.com/Azure/go-autorest/autorest"
-	"github.com/Azure/go-autorest/autorest/azure"
-	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
```

Comment:
@mabhi I noticed that the imports are not properly grouped in this source file and in some others as well. Can you please fix that.

Reply:
Done